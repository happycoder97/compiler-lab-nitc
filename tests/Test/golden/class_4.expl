class A         
{
    int m1() {
        write("M1");
    }

    int m2() {
        write("M2");
    }
}

A a;

int main() {
    initialize();
    a = new(A);
    a.m1();
    a.m2();
    write("DONE");
}
