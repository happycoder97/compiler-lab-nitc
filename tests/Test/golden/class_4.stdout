M1
M2
DONE
